packages/evolution-common/src/services/audits/__tests__/AuditUtils.test.ts

```diff
@@ -74,35 +74,35 @@ describe('auditsToAuditArray', () => {
                 version: 1,
                 errorCode: 'test-error',
                 message: 'Test error message',
-                isWarning: false,
+                level: 'error' as ('error' | 'warning' | 'info' | undefined),
                 ignore: true
             }, 'test-error-2': {
                 version: 1,
                 errorCode: 'test-error-2',
                 message: 'Test error message',
-                isWarning: false,
+                level: 'error' as ('error' | 'warning' | 'info' | undefined),
                 ignore: true
             }
-        }
+        };
         const expectedAudits = [{
             version: 1,
             errorCode: 'test-error',
             message: 'Test error message',
-            isWarning: false,
+            level: 'error' as ('error' | 'warning' | 'info' | undefined),
             ignore: true,
             objectType: 'interview',
             objectUuid: arbitraryUuid
         }, {
             version: 1,
             errorCode: 'test-error-2',
             message: 'Test error message',
-            isWarning: false,
+            level: 'error' as ('error' | 'warning' | 'info' | undefined),
```

Comment:
simply `as const` will compile properly, no need to cast to the whole enum

Reply:
fixed